actions/compose.js

```diff
@@ -0,0 +1,35 @@
+endpoint2action = {
+    action: require("./action").action,
+    click: require("./click").click,
+    fill_form: require("./fill_form").fillForm,
+    back: require("./goback").goBack,
+    forward: require("./goforward").goForward,
+    goto: require("./goto").goto,
+    har: require("./har").har,
+    mhtml: require("./mhtml").captureSnapshot,
+    recaptcha_solver: require("./recaptcha_solver").recaptchaSolver,
+    screenshot: require("./screenshot").screenshot,
+    scroll: require("./scroll").scroll,
+}
+
+async function compose(page, request) {
+    const originalClosePage = request.query.closePage;
+    const originalBody = structuredClone(request.body);
+
+    request.query.closePage = false;
+    delete request.body["actions"];
+
+    let response;
+    try {
+        for (const action of originalBody["actions"]) {
+            Object.assign(request.body, action["body"]);
```

Comment:
- This merges props from different actions and may lead to undesired behavior. For example, consider click followed by scroll without selector.
- This probably won't work with custom js action since its body is not expected to be an object